R/utils.R

```diff
@@ -504,3 +504,97 @@ download_with_retry <- function(...) {
   }
   return(rc)
 }
+
+#' Checks if NONMEM run is done by looking for "Stop Time" in .lst file
+#'
+#' Returns `TRUE` if the model appears to be finished running and `FALSE` otherwise.
+#' @param mod either a bbi_nonmem_model object or a path to an .lst file
+#'
+#' @importFrom readr read_lines
+#'
+#' @export
+check_nonmem_finished <- function(mod) {
```

Comment:
As much as I hate it, you need to add the leading period to this arg (i.e. make it `.mod` instead of `mod`) for consistency's sake. I wish we hadn't committed to this pattern, but at this point there's not really an easy way out of it. And it's better to be annoying and consistent than annoying and inconsistent.

Same is true for all the arguments to `wait_for_nonmem()`.